colors/build.js

```diff
@@ -28,6 +28,12 @@ const jsOutput = [`/* Firefox Colors JS Variables v${metadata.version} */
 
 `];
 
+const gplOutput = [`GIMP Palette
+Name: Firefox/Photon
+# Firefox Colors GPL Color Palette v${metadata.version}
+# ${metadata.homepage}
```

Comment:
Wow, does that work?  Cool!  😃

Reply:
I don't know. Hopefully. Can anyone confirm?

Reply:
(@brassy asked me to test this, as I am a GIMP user)

So the pull request is a step in the right direction, but the array `gplOutput` is never written to a file. @vanillajonathan you'll need another commit that creates the GIMP Palette file. Take a look at the very bottom of the file to see how the JS, CSS and SCSS files are created :)

Reply:
Oops, my bad. I committed a fix for that. Thanks!

Reply:
Great that works. Tiny nit, all other files are named `colors`, yours is `photon`. While I agree that a generic name like `colors` might not be the best, it's best they are all named alike.
So either all colors or photon, or photon-colors..
But this is not my call to make, deferring to @bwinton and @brassy

Reply:
Yeah, I think `colors` would be better for consistency, but after that it looks like it's ready to merge! 🙂

Reply:
Oh, one more thing… Could you add an extra blank line under `# ${metadata.homepage}` to make the output a little prettier? Thanks!

Reply:
Ya know, it's close enough. I'll just fix this up when I merge. 🙂

Reply:
@mozfreddyb The name was `photon.gpl` so it could be directly copied into the palette directory and not collide with any existing palettes.

Reply:
Ah, that avoids a name clash, makes sense. But you can also add the palette file through the UI which makes avoids any name conflict. But as I said..not my decision :)

Reply:
@mozfreddyb The GIMP Palette is compatible with several different software such as GIMP, Inkscape, LibreOffice, Gpick, Just Color Picker, etc. I don't know if all of these support adding the file through UI.